02_cancer_type_classification/domain_adaptation.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exploration of domain adaptation algorithms"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "from tqdm import tqdm\n",
+    "\n",
+    "from transfertools.models import CORAL, TCA\n",
+    "\n",
+    "%reload_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Simulated data\n",
+    "\n",
+    "How do domain adaptation algorithms available in `transfertools` scale with the number of samples/features?"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "def coral_samples(n_samples, n_features, tol=1e-8, seed=1):\n",
```

Comment:
Something went weird in the processing of this chunk, and it's pretty hard to read in the notebook form.

Reply:
Ahh, I had commented out these chunks because they take a while to run. I'll uncomment them and rerun the notebook.